templates/single-quiz/question_type-gap-fill.php

```diff
@@ -35,7 +35,7 @@ class="gapfill-answer-gap" />
 
         <span class="gapfill-answer-post">
 
-            <?php echo apply_filters( 'sensei_answer_text', esc_html(  $question_data[ 'gapfill_post' ]  ) ); ?>
+            <?php echo wp_kses_post( apply_filters( 'sensei_answer_text', $question_data[ 'gapfill_post' ] ) ); ?>
```

Comment:
Could the removal of `esc_html` have backwards compatibility concerns? E.g. if the gapfill question has HTML code that is meant to be escaped and displayed.

Reply:
See https://github.com/Automattic/sensei/pull/2296#discussion_r233540278.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2296/commits/f63c36edc36b683df849bb628b33eafedf4da6ed.